zh/backup-to-gcs.md

```diff
@@ -1,11 +1,12 @@
 ---
-title: 使用 Mydumper 备份 TiDB 集群数据到 GCS
-summary: 介绍如何使用 Mydumper 将 TiDB 集群数据备份到 Google Cloud Storage (GCS)。
+title: 使用 Dumpling 备份 TiDB 集群数据到 GCS
+summary: 介绍如何使用 Dumpling 将 TiDB 集群数据备份到 Google Cloud Storage (GCS)。
+category: how-to
```

Comment:
```suggestion
```
`category` has been deprecated recently. We don't need it in the metadata anymore.